understanding/21/reflow.html

```diff
@@ -38,20 +39,22 @@ <h3>Mobile devices start smaller, how does that work?</h3>
         <h3>Does this mean web sites have to be responsive?</h3>
-        <p>Using responsive design is the most effective method of achieving the goal of allowing people to zoom in to 400%. For organisations which are using legacy systems or are not able to update their layout methods for some reason, an alternative conforming version could be a mobile site which has a fixed 320px wide layout.</p>
+        <p>Using responsive design is the most effective method of achieving the goal of allowing people to zoom in to 400%. For organisations which are using legacy systems or are not able to update their layout methods for some reason, an alternative conforming version could be a mobile site which has a fixed 320px wide layout. The user should be able to find that version from the default website.</p>
         <h3>What happens if sites move/change things at smaller sizes?</h3>
         <p>So long as the information and functionality are still <em>available</em> that is ok. If some content is hidden behind a show/hide button, or navigation is moved into a drop-down it is still available. (Like the 'hamburger' icon pattern for navigation).</p>
         <p>It is possible people might get confused if things move or get hidden, however, the physics demands that something has to move down or be hidden, you cannot make things bigger and fit them in the same space. Also, the people who need it most would typically be zoomed-in when arriving, so may not notice the difference.</p>
         <p>However, if content or functionality is removed that would fail this success criteria.</p>
         <h3>What is the relation to Resize Text?</h3>
         <p>The focus of the Reflow success criteria is to enable users to zoom in without scrolling in two directions. Resize Text (<a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html">Success criterion 1.4.4</a>) also applies, so it should be possible to increase the size of all text by at least 200% <em>as well</em>. If text is reduced in size when people zoom in (or for small-screen usage), it should still be possible to get to 200% bigger. For example, if text is set at 20px when the window is 1280px wide, at 200% zoom it should be at least 20px (so 200% larger), but at 400% zoom it could be set to 10px (therefore still 200% larger than the 100% view. It is not required to be 200% larger at every breakpoint, but it should be possible to get 200% large text in some way.</p>
+        <h3>Does every variation of the page have to conform?</h3>
+        <p><a href="https://www.w3.org/TR/WCAG21/#h-note-27">Conformance for WCAG 2.1</a> is for full pages including variations (at the same URL), so each variation (break point in CSS) should conform. The amount of work involved is greatly reduces if you know what changes happen at each variation.</p>
```

Comment:
"The amount of work involved is greatly reduces..." > "The amount of work involved is greatly reduced..."